ste/folderCreationTracker.go

```diff
@@ -85,6 +85,11 @@ func (f *jpptFolderTracker) CreateFolder(folder string, doCreation func() error)
 		return nil // Never persist to dev-null
 	}
 
+	if idx, ok := f.contents[folder]; ok &&
```

Comment:
is there any way we could add some way to test that the minimal number of and expected calls are made to folder create? Perhaps through a mock folder creation tracker in test?